provision/main.yaml

```diff
@@ -0,0 +1,107 @@
+---
+- name: Perform common tasks for all hosts
+  hosts: all
+  tasks:
+      - name: Disable requiretty in sudoers
+        lineinfile: dest=/etc/sudoers state=absent regexp="^Defaults requiretty.*"
+        become: true
+
+      - name: Ensure epel is enabled when enable_epel is True
+        yum: name=epel-release state=present
+
+      - name: yum update all
+        yum: name='*' state=latest
+
+      - name: SELinux permissive
+        selinux: policy=targeted state=permissive
+
+      - name: Install packages
+        yum: package={{ item }} state=latest
+        with_items:
+            - NetworkManager
+            - firewalld
+            - python-rhsm-certificates
+            - centos-release-openshift-origin39.noarch
+            - python-ipaddress
+            - docker
+            - rsync
+            - PyYAML
+
+      - name: Start services
+        systemd: state=started enabled=yes name={{ item }}
+        with_items:
+            - NetworkManager
+            - firewalld
+            - docker
```

Comment:
Why are we starting Docker service here? That's something that `prerequisistes.yml` does take care of for us. I'd suggest to remove it from here.

Reply:
removed

Reply:
Not removed!

Reply:
pr #42 this is part of provisioning